extensions/amp-web-vitals/0.1/test/validator-amp-web-vitals.html

```diff
@@ -0,0 +1,27 @@
+<!doctype html>
```

Comment:
This needs a copyright declaration at the top of the file as well as a short description. The `.out` file will need to be regenerated afterwards.

Reply:
Hi, Sorry I opened this PR by mistake as I am doing an experiment with a new component for the time being for a project. I might open a proper PR at some point and will take your good feedback onboard :)